src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -72,8 +79,7 @@ services:
       - '@prestashop.adapter.product.image.product_image_url_factory'
       - '@prestashop.adapter.product.combination.repository.combination_repository'
 
-  prestashop.adapter.product.image.repository.product_image_multishop_repository:
```

Comment:
Is this a new service ? otherwise you can't rename its definition

Reply:
It's a service which I added in a previous PR only in develop. So no module use it and I changed the necessary in the Core (just 2 or 3 dependencies).

Reply:
Any service introduced for the product page V2 can be renamed as they are all experimental so far